src/transactions/SponsorshipUtils.cpp

```diff
@@ -198,6 +199,10 @@ computeMultiplier(LedgerEntry const& le)
     case OFFER:
     case DATA:
         return 1;
+#ifdef ENABLE_NEXT_PROTOCOL_VERSION_UNSAFE_FOR_PRODUCTION
+    case CONTRACT_DATA:
+        return 1;
+#endif
```

Comment:
No, in consultation with @jonjove this is an unreachable code path for CONFIG.

Reply:
I asked @jonjove again about this today (sorry, there are a lot of cases to handle and this one keeps coming up because it's weird) and he suggested I can remove _both_ here and let them both fail because they should be rejected in `RevokeSponsorshipOpFrame::doCheckValid`. So I'll remove this.